pyserini/search/pysearch.py

```diff
@@ -71,6 +71,7 @@ class SimpleSearcher:
 
     def __init__(self, index_dir: str):
         self.object = JSimpleSearcher(JString(index_dir))
+        self.maxDoc = self.object.getTotalNumDocuments()
```

Comment:
A variable name that is more pythonic? How about `number_of_docs` or even just `num_docs`?